spec/cluster_spec.rb

```diff
@@ -3,6 +3,88 @@
 require "spec_helper"
 require "cluster"
 
+class Synchronizer
```

Comment:
All the changes here reflect stuff that used to be in `ClusterGetter`

Reply:
My main concern here is that I don't understand `InstrumentedCluster` well enough to tell if this is testing conditions similar to what we expect to see in production. I can be OK with this for now, as long as I get to spend some time with this later.